twa

```diff
@@ -390,8 +390,22 @@ function stage_4_scm_repos {
   done
 }
 
+# Stage 5: Check if the server provides a robots.txt file.
+#
+# Check for "The Robots Exclusion Protocol".
+# Will follow redirects to ensure file exists.
+function stage_5_robots_check {
+  verbose "Stage 5: Robots Exclusion Protocol check"
+
+  http_response_code=$(curl -L -s -o /dev/null -w "%{http_code}" "${domain}/robots.txt")
+  if [[ ${http_response_code} == 200 ]]; then
+    PASS "Robots.txt is provided at: ${domain}/robots.txt"
+  else
```

Comment:
Let's be explicit here, and check for a 404 for the `MEH`. Then, the base `else` case can be an `UNK`.

Reply:
(See `stage_4_scm_repos` for an example of this).